tests/functional/complete_sign_up.js

```diff
@@ -5,50 +5,45 @@
 define([
   'intern',
   'intern!object',
-  'require',
-  'intern/browser_modules/dojo/node!xmlhttprequest',
-  'app/bower_components/fxa-js-client/fxa-client',
   'app/scripts/lib/constants',
-  'tests/lib/restmail',
   'tests/lib/helpers',
   'tests/functional/lib/helpers'
-], function (intern, registerSuite, require, nodeXMLHttpRequest, FxaClient, Constants, restmail, TestHelpers, FunctionalHelpers) {
+], function (intern, registerSuite, Constants, TestHelpers, FunctionalHelpers) {
   var config = intern.config;
-  var AUTH_SERVER_ROOT = config.fxaAuthRoot;
-  var EMAIL_SERVER_ROOT = config.fxaEmailRoot;
   var PAGE_URL_ROOT = config.fxaContentRoot + 'verify_email';
   var PASSWORD = 'password';
-  var user;
   var email;
   var accountData;
-  var client;
   var code;
   var uid;
 
+  var click = FunctionalHelpers.click;
   var createRandomHexString = TestHelpers.createRandomHexString;
+  var createUser = FunctionalHelpers.createUser;
   var fillOutSignUp = FunctionalHelpers.fillOutSignUp;
+  var getVerificationLink = FunctionalHelpers.getVerificationLink;
   var noSuchElement = FunctionalHelpers.noSuchElement;
+  var openPage = FunctionalHelpers.openPage;
+  var testElementExists = FunctionalHelpers.testElementExists;
   var testSuccessWasShown = FunctionalHelpers.testSuccessWasShown;
+  var visibleByQSA = FunctionalHelpers.visibleByQSA;
```

Comment:
When so many of these are changing at once, maybe it's a good moment to switch to destructuring assignment?